text/survey_analysis_report2.md

```diff
@@ -0,0 +1,219 @@
+
+## GIN-tonic implementation
+
+GIN is the G-Node infrastructure.
+It is based on gogs, git and git annex technologies and
+ brings non only most of the project management and coordination tools that made the success of open source software development,
+but also large file support and data publication.
+It is compatible with the git submodule technology, where sub-folders can be synchronized, shared and published independently of the other sub-folders,
+while looking completely normal on one's computer.
+GIN brings therefore the possibility to publish sub-modules independently of each others, 
+which will ease the opening of research data.
+It might also make [the use of markdown and latex to write manuscripts](https://oa-pub.hos.tuhh.de/en/project/) a straightforward choice, as these technologies can use git as a version control system.
+
+
+We are building an extension that will facilitate some administrative tasks and automate some complex work-flows linked to the use of the template.
+We could not resist calling it tonic, in reference to the vigor added to the GIN tool. The tonic tool is still in development, and we we are also working on the implementation of the tonic concept for GitLab-based platforms (that is, using git and git-LFS), called LAB-tonic. 
+
+  
+
+The tonic application creates a new project repository,
+cloning the research folder structure (with some folders being created as sub-modules), and adding a script that will synchronize the repository and its sub-modules on a double click. It also adds some submodules (shared-figures, manuscripts, and report_conf) to loboratory-wide repositories, automatically allowing laboratory level organisation of some files.
+This means that for example a manuscript draft can be available in the project folder on the student computer, while the same data will be available in the lab_manuscript folder on the PI computer, both version being synchronized with a unique version on the GIN server. 
+
+
+
+
+On the other hand, the tonic application will also be able to add submodules and folders to the parent repository. For each experiment performed, a data submodule  will be created, so that the data can be curated and published independently of the other experiments, while other normal folders will be created (see Fig. 4). A synchronisation of the computer version will then bring these changes on the local version, showing the new folders ready to be filled with data, code, and figures.
```

Comment:
```suggestion
In addition, the tonic application will also be able to add submodules and folders to the parent repository. For each experiment performed, a data submodule  will be created, so that the data can be curated and published independently of the other experiments, while other normal folders will be created (see Fig. 4). A synchronisation of the computer version will then bring these changes on the local version, showing the new folders ready to be filled with data, code, and figures.
```